src/ucs/type/spinlock.h

```diff
@@ -40,50 +51,71 @@ typedef struct ucs_recursive_spinlock {
 } ucs_recursive_spinlock_t;
 
 
-static ucs_status_t ucs_spinlock_init(ucs_spinlock_t *lock, int flags)
+void ucs_recursive_spinlock_destroy(ucs_recursive_spinlock_t *lock);
+
+/**
+ * Spinlock implementation section
+ */
+static ucs_status_t ucs_spinlock_init(ucs_spinlock_t *lock)
```

Comment:
UCS_F_ALWAYS_INLINE
or compiler can complain about unused

also, maybe make it void?

Reply:
declared as UCS_F_ALWAYS_INLINE void ...